polytope_server/common/datasource/polytope.py

```diff
@@ -22,18 +22,16 @@
 import logging
 import os
 import subprocess
-
-os.environ["GRIBJUMP_HOME"] = "/opt/fdb-gribjump"
-
 import tempfile
 from pathlib import Path
 
-import polytope
 import yaml
 
 from ..caching import cache
 from . import datasource
 
+os.environ["GRIBJUMP_HOME"] = "/opt/fdb-gribjump"
```

Comment:
I think this is here, along with the `import polytope` to pre-seed pygribjump module with the correct config. Would be best to revert this and add PEP ignores for now. I think there is a PR in gribjump to accept the config at construction which we could use instead (soon).

Reply:
should be good to go now